pkg/models/model_image.go

```diff
@@ -15,10 +15,12 @@ type Image struct {
 	ID int `json:"id"`
 
 	Title     string `json:"title"`
+	URL       string `json:"url"`
 	Rating    *int   `json:"rating"`
 	Organized bool   `json:"organized"`
 	OCounter  int    `json:"o_counter"`
 	StudioID  *int   `json:"studio_id"`
+	Date      *Date  `json:"date"`
```

Comment:
You need to add a `Date()` resolver method to `api.imageResolver` for this field. See `Date()` in `api/resolver_model_scene.go`.

This explains the following error:
```
D:\Projets\stashStar\internal\api\resolver.go:63:9: cannot use &(imageResolver literal) (value of type *imageResolver) as ImageResolver value in return statement: *imageResolver does not implement ImageResolver (missing method Date)
```